src/screens/FeedScreen/ContentPoll.js

```diff
@@ -131,7 +132,7 @@ const stylesComponent = (pollLength) =>
       height: 4,
       borderRadius: 4,
       alignSelf: 'center',
-      backgroundColor: COLORS.blackgrey
+      backgroundColor: colors.blackgrey
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color property has been updated from `COLORS.blackgrey` to `colors.blackgrey`. Make sure that `colors.blackgrey` is defined and is the intended color.
```diff
- backgroundColor: COLORS.blackgrey
+ backgroundColor: colors.blackgrey
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->